tests/test_livysession.py

```diff
@@ -436,6 +437,21 @@ def test_create_sql_hive_context_happens_once(self):
                in http_client.post_statement.call_args_list
         assert len(http_client.post_statement.call_args_list) == 1
 
+        session._get_sql_context_creation_command = MagicMock()
+
+        session._get_sql_context_creation_command.return_value.execute.return_value = (True, "Success")
+        session.created_sql_context = None
+        session.create_sql_context()
+        assert session.created_sql_context
+
+        session._get_sql_context_creation_command.return_value.execute.return_value = (False, "Exception")
+        session.created_sql_context = None
+        try:
+            session.create_sql_context()
```

Comment:
The change you made wasn't really necessary, just `assert False` at this location (like I suggested) to make sure an exception was thrown